src/blocks/accordion/accordion-item/styles/editor.scss

```diff
@@ -76,3 +76,14 @@
 [data-type="coblocks/accordion-item"]:not(.is-selected) > .block-editor-block-list__block-edit::before {
 	border: 1px dashed rgba(123, 134, 162, 0.3);
 }
+
+.dashicons {
```

Comment:
This would affect global styles for the class `.dashicons`. This CSS must live within the selector hierarchy for the `.wp-block-coblocks-accordion-item` to be properly scoped. Look at line 2 of this file.

Reply:
these styles have been moved in the newest commit of this branch - https://github.com/godaddy-wordpress/coblocks/blob/issue-1914/src/blocks/accordion/accordion-item/styles/editor.scss